src/Pagination/PaginationManipulator.php

```diff
@@ -72,6 +72,10 @@ protected function registerConnection(
         int $maxCount = null,
         ObjectTypeDefinitionNode $edgeType = null,
     ): void {
+        if (! isset($this->documentAST->types[self::pageInfo()->getName()->value])) {
+            $this->documentAST->setTypeDefinition(self::pageInfo());
```

Comment:
Extract the duplicated call to `self::pageInfo()`, same with the other places like this.